osu.Game/Overlays/MusicController.cs

```diff
@@ -24,7 +25,7 @@ public class MusicController : Component, IKeyBindingHandler<GlobalAction>
         [Resolved]
         private BeatmapManager beatmaps { get; set; }
 
-        private List<BeatmapSetInfo> beatmapSets;
+        public readonly BindableList<BeatmapSetInfo> BeatmapSets = new BindableList<BeatmapSetInfo>();
```

Comment:
This should at the very least be an IBindableList if this is how this is going to be done.